Lombiq.HelpfulLibraries.OrchardCore/TagHelpers/EditorFieldSetTagHelper.cs

```diff
@@ -52,11 +52,27 @@ public override Task ProcessAsync(TagHelperContext context, TagHelperOutput outp
             output.Attributes.Add(Class, "form-group mb-3 col-xl-6");
         }
 
-        var isRequired = IsRequired || HasRequiredAttribute(For);
-        var isCheckbox = InputType.EqualsOrdinalIgnoreCase("checkbox");
+        AppendInputAndLabel(
+            output,
+            InputType.EqualsOrdinalIgnoreCase("checkbox"),
+            IsRequired || HasRequiredAttribute(For));
 
-        if (isCheckbox) output.Content.AppendHtml("<div class=\"custom-control custom-checkbox\">");
+        var tagBuilder = _htmlGenerator.GenerateValidationMessage(
+            ViewContext,
+            For.ModelExplorer,
+            For.Name,
+            message: null,
+            tag: "span",
+            htmlAttributes: null);
+        AppendContent(output, tagBuilder);
```

Comment:
While at times having an extra local variable **does** increase readability, I'd argue that in this case the code is more readable without it:
```suggestion
        AppendContent(output, _htmlGenerator.GenerateValidationMessage(
            ViewContext,
            For.ModelExplorer,
            For.Name,
            message: null,
            tag: "span",
            htmlAttributes: null));
```

Reply:
Hard disagree, having an inner multiline argument is bad. At a glance it looks like those in the indented lines are the arguments of the outer method. In this case it even obscures the important context of _what_ is being added.

Reply:
While I don't necessarily agree with the multiline argument, I actually did hesitate on the _what_ part, and agree with your disagreement.

Reply:
LGTM.